pkg/volume/vspherevolume/vsphere_volume_util.go

```diff
@@ -63,10 +64,13 @@ const (
 	IopsLimitCapabilityMin              = 0
 )
 
+// ErrProbeVolume is an error instance for failed scanning of attached volumes.
 var ErrProbeVolume = errors.New("Error scanning attached volumes")
```

Comment:
This variable is never used anywhere so we can drop it (at least I couldn't find any usage with grep and my IDE complains).

Reply:
Yes i think you are right.